src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Call.cs

```diff
@@ -652,12 +654,21 @@ private static ImmutableArray<RefKind> GetEffectiveArgumentRefKinds(ImmutableArr
             if (!ignoreComReceiver)
             {
                 var receiverNamedType = invokedAsExtensionMethod ?
-                                        ((MethodSymbol)methodOrIndexer).Parameters[0].Type.TypeSymbol as NamedTypeSymbol :
+                                        @params[0].Type.TypeSymbol as NamedTypeSymbol :
                                         methodOrIndexer.ContainingType;
-            isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+                isComReceiver = (object)receiverNamedType != null && receiverNamedType.IsComImport;
+            }
+
+            if (!@params.IsDefaultOrEmpty)
```

Comment:
>IsDefaultOrEmpty [](start = 25, length = 16)

nit: Technically this could just be `IsEmpty` as the doc for `MethodSymbol.Parameters` says it returns empty array when no parameters, so the same applies to `GetParameters()` on a method symbol.

nit: consider adding a comment on the `return false;`

Reply:
Dug back into this and am thinking this doesn't actually make sense. This method is used in more contexts than just lowering of collection initializers and wasn't actually preventing the lowering of the BoundCollectionElementInitializer as I thought it was earlier. I've reverted back to the master version of this file and all the tests are continuing to pass.